### PR TITLE
[SpritePlugin] Added support for scaling and transform-origin

### PR DESCRIPTION
**Pull request by sampi**
@wiredearp @zdlm @sampi

Fixes issue #303 - Added support for `scale()` and `transform-origin` in the SpritePlugin.

Other changes:
* Upgraded Jasmine to 2.6.4
  * [2.6.4](https://github.com/jasmine/jasmine/blob/2_6/release_notes/2.6.4.md)
  * [2.6.3](https://github.com/jasmine/jasmine/blob/2_6/release_notes/2.6.3.md)
  * [2.6.2](https://github.com/jasmine/jasmine/blob/2_6/release_notes/2.6.2.md)
* Upgraded all dependencies
  * Cheerio will be permanently pegged to `0.22.0` until they fix this [issue](https://github.com/cheeriojs/cheerio/issues/1042)
* Reformatted the code according to the latest Prettier ruleset

### CODE REVIEW

File: docs/dist/appendix/autolayout/index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
 <body data-ts="Spirit" id="body">
     <script class="gui-script" data-ts.scriptid="edbml.$iSNtu1">
         edbml.declare("edbml.$iSNtu1").as(function $edbml(
-            /**/
+            /*``*/
```
Comment (wiredearp) [inline docs/dist/appendix/autolayout/index.html]: This is getting more and more strange :cry:

Comment (sampi) [inline docs/dist/appendix/autolayout/index.html]: yea, I'm not sure where this came from =/

**Comment by sampi** [comment]
@wiredearp I've removed the Prettier changes.
the new version was 1.4.4, which was within the range for the semver in package.json, that's why it wasn't in there.

**Comment by sampi** [comment]
retest

**Pull request merged**